app/components/UserAttendance/AttendanceModal.tsx

```diff
@@ -47,11 +50,13 @@ const Tab = ({ name, index, activePoolIndex, togglePool }: TabProps) => (
 
 type State = {
   pools: Pool[];
+  filter: string;
 };
 
 class AttendanceModal extends Component<Props, State> {
   state = {
     pools: [],
+    filter: '',
```

Comment:
I would change the name of this variable to `search` or something similar since that is what it actually does. `filter` makes it seem like it is filtering out users in certain grades and such, not displaying users with a matching name.

Reply:
It's literally filtering out user from the list though, and I chose it since `filter` is also used in the table column search.